code/modules/ninja/suit/suit_process.dm

```diff
@@ -1 +1,20 @@
-// TODO
\ No newline at end of file
+/obj/item/clothing/suit/space/space_ninja/proc/ntick(mob/living/carbon/human/U = affecting)
+	//Runs in the background while the suit is initialized.
+	//Requires charge or stealth to process.
+	spawn while(s_initialized)
```

Comment:
I know you said no styling comments but jesus christ
Spawn is a block level proc, it shouldn't have anything on the same line as it
```
spawn()
    while()
```